docs/_importers/blogger.md

```diff
@@ -1,29 +1,13 @@
 ---
-layout: docs
-title: Blogger
 importer: true
 prev_section: behance
 link_source:  blogger
```

Comment:
Side note: could we lexically alphabetize the list of importers and set these in a plugin or in Liquid too? This seems a bit inefficient to list them all here. When adding a new importer, the contributor has to know the order of their new importer in the context of other importers which we could easily get a computer to figure that out.

Reply:
Technically, this is not needed at all. At least with modern Jekyll because basically there's already the concept of `next_doc` and `prev_doc` for `Jekyll::Document` objects.
These were removed in the other pull request that made wholesale changes to the docs but left it untouched here to maintain the narrow scope of current PR.